src/preamble.js

```diff
@@ -1764,6 +1764,7 @@ var Math_imul = Math.imul;
 var Math_fround = Math.fround;
 var Math_min = Math.min;
 var Math_clz32 = Math.clz32;
+var Math_trunc = Math.trunc;
```

Comment:
I had to add this line, otherwise I got an error because `Math_trunc` was not found or something like this.